lib/components/narrative/default/default-itinerary.js

```diff
@@ -1,34 +1,76 @@
+import moment from 'moment-timezone'
 import coreUtils from '@opentripplanner/core-utils'
 import React from 'react'
+import { FormattedMessage, FormattedNumber, IntlProvider } from 'react-intl'
+import { connect } from 'react-redux'
+import { getUrlParams } from '@opentripplanner/core-utils/lib/query'
 
 import NarrativeItinerary from '../narrative-itinerary'
 import ItineraryBody from '../line-itin/connected-itinerary-body'
-import ItinerarySummary from './itinerary-summary'
 import SimpleRealtimeAnnotation from '../simple-realtime-annotation'
-import { getTotalFareAsString } from '../../../util/state'
+import { getMessages } from '../../../util/i18n'
+
+import ItinerarySummary from './itinerary-summary'
 
-const { isBicycle, isTransit } = coreUtils.itinerary
-const { formatDuration, formatTime } = coreUtils.time
+const { calculateFares, isBicycle, isTransit } = coreUtils.itinerary
 
+/**
+ * Obtains the description of an itinerary in the given locale.
+ */
 // FIXME move to core utils
 function getItineraryDescription (itinerary) {
   let primaryTransitDuration = 0
-  let mainMode = 'Walk'
+  let mainModeStringId = 'walk'
   let transitMode
   itinerary.legs.forEach((leg, i) => {
     const {duration, mode, rentedBike} = leg
     if (isTransit(mode) && duration > primaryTransitDuration) {
       // TODO: convert OTP's TRAM mode to the correct wording for Portland
       primaryTransitDuration = duration
-      transitMode = mode.toLowerCase()
+      transitMode = <FormattedMessage id={mode} />
     }
-    if (isBicycle(mode)) mainMode = 'Bike'
-    if (rentedBike) mainMode = 'Bikeshare'
-    if (mode === 'CAR') mainMode = 'Drive'
+    if (isBicycle(mode)) mainModeStringId = 'bike'
+    if (rentedBike) mainModeStringId = 'bikeshare'
+    if (mode === 'CAR') mainModeStringId = 'drive'
```

Comment:
needs an entry for Micromobility.

Reply:
Was updated in https://github.com/opentripplanner/otp-react-redux/pull/375/commits/d0f962fcf847a7ba0d3b9a9b9fa33501ce41e00a.